TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+       "                            \n",
+       "var gd = document.getElementById('a5e238be-70e3-49c9-b8ae-9a0a49bcc0e6');\n",
+       "var x = new MutationObserver(function (mutations, observer) {{\n",
+       "        var display = window.getComputedStyle(gd).display;\n",
+       "        if (!display || display === 'none') {{\n",
+       "            console.log([gd, 'removed!']);\n",
+       "            Plotly.purge(gd);\n",
+       "            observer.disconnect();\n",
+       "        }}\n",
+       "}});\n",
+       "\n",
+       "// Listen for the removal of the full notebook cells\n",
+       "var notebookContainer = gd.closest('#notebook-container');\n",
+       "if (notebookContainer) {{\n",
+       "    x.observe(notebookContainer, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "// Listen for the clearing of the current output cell\n",
+       "var outputEl = gd.closest('.output');\n",
+       "if (outputEl) {{\n",
+       "    x.observe(outputEl, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "                        })                };                });            </script>        </div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "broadway['log_avg_ticket_price'] = np.log(broadway['avg_ticket_price'])\n",
+    "broadway['weekly_gross_overall_in_100k'] = broadway['weekly_gross_overall']/100000 \n",
+    "fig = px.scatter(broadway, x='weekly_gross_overall_in_100k', \n",
+    "                 y='log_avg_ticket_price', trendline='ols')\n",
+    "fig.show()"
+   ]
```

Comment:
This is the last bit of code that I'm "taking under consideration.

Reply:
You'll be replacing this with a demo of "formula" based `statsmodels` model fitting and hypothesis testing